README.md

```diff
@@ -1,5 +1,33 @@
-# postgresql-bundle
+# PostgreSQL Bundle
 
-A juju bundle for deploying PostgreSQL and PGBouncer VM charms.
+A juju bundle for deploying PostgreSQL and PGBouncer machine charms, with TLS enabled by default. For kubernetes, see the [postgresql-k8s-bundle](https://github.com/canonical/postgresql-k8s-bundle).
 
-NB: The TLS implementation is self-signed - if other certificates are required, please follow the [TLS Certificate Operator Documentation](https://charmhub.io/tls-certificates-operator) to upload your own.
+Included charms:
+- [postgresql-operator](https://github.com/canonical/postgresql-operator)
+  - 2 replica units
+  - related to pgbouncer-operator for connection pooling
+  - related to tls-certificates-operator for TLS.
+- [pgbouncer-operator](https://github.com/canonical/pgbouncer-operator)
+  - Note: TLS not implemented in this proxy yet
+- [tls-certificates-operator](https://github.com/canonical/tls-certificates-operator)
+  - The TLS implementation is self-signed - if other certificates are required, please follow the [TLS Certificate Operator Documentation](https://charmhub.io/tls-certificates-operator) to upload your own.
+
+## Deployment
+
+To deploy in a VM juju model:
+```bash
+juju deploy postgresql-bundle
+juju add-relation pgbouncer:database my-app:backend-database
```

Comment:
It would be good to follow the same pattern that is used in the [k8s bundle](https://github.com/canonical/postgresql-k8s-bundle).

Reply:
Yeah, I planned to publish it here too, but need a time to prepare and polish. :-D